src/backend/src/controllers/change-requests.controllers.ts

```diff
@@ -9,7 +9,14 @@ import { Role } from '@prisma/client';
 
 export const getAllChangeRequests = async (req: Request, res: Response) => {
   const changeRequests = await prisma.change_Request.findMany(changeRequestRelationArgs);
-  return res.status(200).json(changeRequests.map(changeRequestTransformer));
+  try {
+    return res.status(200).json(changeRequests.map(changeRequestTransformer));
+  } catch (e) {
+    if (e instanceof TypeError) {
+      return res.status(400).json({ message: e.message });
```

Comment:
I don't think 400 is necessarily the appropriate code

Reply:
400 indicates client error, but I don't super recall precisely what error states are possible